### ISSUE TITLE
refactor to use object oriented methods: `band.openings` and `band.members`

### ISSUE THREAD

**Issue by nathanallen** [LilyCole/bandapp#32]
`get_openings` and `get_members` feels like the kind of thing a `band` knows about the world.

Refactor to use object oriented methods: `band.openings` and `band.members`

See [bands_controller#L69-L75](https://github.com/LilyCole/bandapp/blob/c030da5bdc9770c04caeef15d550c92c294a82ba/app/controllers/bands_controller.rb#L69-L75) & [band_instrument_musicians_controller.rb#L41-L43](https://github.com/LilyCole/bandapp/blob/c030da5bdc9770c04caeef15d550c92c294a82ba/app/controllers/band_instrument_musicians_controller.rb#L41-L43)

**Comment by Lily-Brown** [comment]
Fixes issue in Pull Request #42 for Band Controller. Band_Instrument_Musician Controller remains same.

**Comment by nathanallen** [comment]
Naming things is hard! As submitted your `members` is actually `memberships` (e.g. band_instrument_musicans instead of musicians). Which do we want?

**Comment by nathanallen** [comment]
Refactor:

``` ruby
  # /models/band.rb
  def openings
    BandInstrumentMusician.all.where({musician_id: nil})
  end
```

``` ruby
  # /bands_controller.rb
     @openings = @band.openings.all.where(band_id: @band.id)
```

Merged together this becomes:

``` ruby
BandInstrumentMusician.all.where({musician_id: nil}).all.where(band_id: @band.id)
```

Which in translation is:

``` ruby
# for a given band, find me all the openings:
band.band_instrument_musicians.where({musician_id: nil})
```

**Comment by Lily-Brown** [comment]
@nathanallen Yes!! We were having trouble chaining those together before! Thanks for the help! Works great!
